assets/src/edit-story/components/canvas/layout.js

```diff
@@ -123,7 +123,7 @@ const PageAreaSafeZone = styled.div`
 
 const HeadArea = styled(Area).attrs({ area: 'head', overflowAllowed: false })``;
 
-const MenuArea = styled(Area).attrs({ area: 'menu', overflowAllowed: false })``;
+const MenuArea = styled(Area).attrs({ area: 'menu', overflowAllowed: true })``;
```

Comment:
Hm, what's the overflow for?

Reply:
The focus border overflows. Without it you get:

![Screen Shot 2021-02-09 at 17 22 01](https://user-images.githubusercontent.com/637548/107436341-647a3c00-6afb-11eb-84d2-b5cbf2edce09.png)

Which looks stupid 🤷 . And I don't think the overflow here hurts.